### DISCUSSION TITLE
post/2023-03-06-workingcopy-adds-openai-generated-commit-messages/

### DISCUSSION THREAD

**Discussion by giscus** [marcusramberg/blog#7] [Announcements]
# post/2023-03-06-workingcopy-adds-openai-generated-commit-messages/

Making (things|code|mischief|mistakes).

https://marcus.means.no/post/2023-03-06-workingcopy-adds-openai-generated-commit-messages/

**Comment by marcusramberg** [comment]
So, I heard you have comments now, _Father_?
